src/app/services/submissions.service.ts

```diff
@@ -40,12 +45,27 @@ export class SubmissionsService {
   /**
    * List Projects for Current Logged in user.
    */
-  getDataTypes(token: String) {
-    let requestUrl =  this.dataTypesEndpoint;
-    var response = this.http.get(requestUrl);
+  getSubmissionPlansResponse(token: String) {
+
+    const requestUrl =  this.SubmissionPlansEndpoint;
+    const response = this.http.get(requestUrl);
     return response;
   }
 
+  getSubmissionPlansUIData(submissionPlans) {
+    let submissionPlansUIData = [];
+    for (const submissionPlan of submissionPlans) {
+      const submissionPlanUIData = {};
```

Comment:
The choice of `let` vs. `const` doesn't affect memory or performance. Both choices compiles to the [same code](https://bit.ly/2QvSiMV) . However, you are more strictly informing the ts compiler about your intentions. If you have no intent to mutate the submissionPlan variable, then I would suggest is better to stick to `const`.

Reply:
yes. the same. The lint suggested for me to use `const`, so I changed it.

Reply:
Yea I was just doing a search for const and I found out that its a block-scoped similar to let. So yea in this case it will not save any extra memory.